server/config.example.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright © 2023 OpenIM open source community. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+app:
+  version: 0.1.0-dev
+  debug: true
+  log_file: logs/
+
+server:
+  ip: 127.0.0.1
+  port: 10010
+
+jwt:
+  secret: openkf
+  issuer: openkf
+  expire_days: 1
+
+mysql:
+  ip: 127.0.0.1
+  port: 3307
```

Comment:
we use 3307 here? I remember mysql default port is 3306

Reply:
The agreed 3307

Reply:
We're forwarding ports here.